### ISSUE TITLE
how to get logits (softmax) along with the audio sample while doing inference?

### ISSUE THREAD

**Issue by sameerkhurana10** [NVIDIA/nv-wavenet#61]

**Comment by naba89** [comment]
Hey @sameerkhurana10 ,

Did you figure this out? Any leads?
